tests/foreman/cli/test_user.py

```diff
@@ -448,3 +480,26 @@ def test_expired_personal_access_token(self):
 
         :CaseImportance: Medium
         """
+        user = make_user()
+        User.add_role({'login': user['login'], 'role': 'View hosts'})
+        token_name = gen_alphanumeric()
+        datetime_now = datetime.datetime.utcnow()
+        datetime_expire = datetime_now + datetime.timedelta(seconds=20)
```

Comment:
In Test step, it is mentioned 1 minute?

Reply:
Ah my bad. When I wrote the stubbed cases I thought minimum is 1 minute. But while writing it I came across seconds.